src/frontends/lean/elaborator.cpp

```diff
@@ -2766,7 +2779,9 @@ static expr translate(environment const & env, local_context const & lctx, expr
 
 expr nested_elaborate(environment & env, options const & opts, metavar_context & mctx, local_context const & lctx,
                       expr const & e, bool relaxed) {
-    elaborator elab(env, opts, mctx, lctx);
+    // TODO(sullrich): pass through info_manager?
```

Comment:
I wasn't sure whether the `info_manager` would have to be part of the elaborator snapshots, could we get redundant/wrong information when using a global one?